### PR TITLE
843009-Prepare UG Documentation for Directional Zooming Support in .NET MAUI

### PR DESCRIPTION
**Pull request by DhanarajRajendran**
Added the Used guide for Directional Zooming support in .NET MAUI.

### CODE REVIEW

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 1]
```diff
@@ -79,6 +79,43 @@ chart.ZoomPanBehavior = zooming;
 
 {% endtabs %}
 
+### Directional Zooming
+
+The directional Zooming feature enhances your zooming experience by allowing you to zoom in and out in a specific direction. This feature is enabled by setting the [EnableDirectionalZooming]() property to `true` as shown in the following code sample and the [EnablePinchZooming](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartZoomPanBehavior.html#Syncfusion_Maui_Charts_ChartZoomPanBehavior_EnablePinchZooming) should not be set as `false,` because it relies on the pinch gesture direction. The default value of this property is false.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:84]: EnablePinchZooming property should be set as true. Update it in tips

Comment (DhanarajRajendran) [reply MAUI/Cartesian-Charts/Zooming-and-panning.md]: Update the changes and add it in tips.

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 2]
```diff
@@ -79,6 +79,45 @@ chart.ZoomPanBehavior = zooming;
...
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    ...
+    ChartZoomPanBehavior zooming = new ChartZoomPanBehavior()
+    {
+        EnablePinchZooming=true,
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:107]: `EnablePinchZooming=true,`

Alignment not good.

Comment (DhanarajRajendran) [reply MAUI/Cartesian-Charts/Zooming-and-panning.md]: Addressed and Updated the changes.

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine.

**Pull request merged**